design-system/skyyrose-storefront/src/tokens/tokens.css

```
@@ -0,0 +1,326 @@
+	--color-gold-rgb:  212, 175, 55;
+	--color-crimson:   var(--crimson, #DC143C);
+	--color-silver:    var(--silver, #C0C0C0);
+	--color-white:     var(--white, #FFF);
+	--color-black:     var(--black, #000);
+
+	/* --- §1.6  Color — Semantic surfaces --------------------------- */
+	--color-page-bg:       #0A0A0A;
+	--color-card-bg:       #111;
+	--color-card-bg-hover: #161616;
+	--color-border:        #2A2A2A;
+	--color-border-hover:  rgba(183, 110, 121, 0.3);
```

Stylelint (17.13.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)